pybullet-helpers/src/pybullet_helpers/utils.py

```diff
@@ -122,6 +163,180 @@ def create_pybullet_cylinder(
     return cylinder_id
 
 
+def get_triangle_vertices(
+    triangle_type: str,
+    side_lengths: tuple[float, float],
+) -> list[list[float]]:
+    """Get the vertices of a triangle given its type and side lengths."""
+
+    vertices = []
+    if triangle_type == "equilateral":
+        side = side_lengths[0]
+        height = (np.sqrt(3) / 2) * side
+        v0 = [-side / 2, -height / 3, 0]
+        v1 = [side / 2, -height / 3, 0]
+        v2 = [0, 2 * height / 3, 0]
+        vertices = [v0, v1, v2]
+    elif triangle_type == "isosceles":
+        base = side_lengths[0]
+        height = side_lengths[1]
+        v0 = [-base / 2, 0, 0]
+        v1 = [base / 2, 0, 0]
+        v2 = [0, height, 0]
+        vertices = [v0, v1, v2]
+    elif triangle_type == "right":
+        base = side_lengths[0]
+        height = side_lengths[1]
+        v0 = [0, 0, 0]
+        v1 = [base, 0, 0]
+        v2 = [0, height, 0]
+        vertices = [v0, v1, v2]
+    else:
+        raise ValueError(f"Unknown triangle type: {triangle_type}")
+
+    return vertices
+
+
+def create_pybullet_triangle(
+    color: tuple[float, float, float, float],
+    triangle_type: str,
+    side_lengths: tuple[float, float],
+    depth: float,
+    physics_client_id: int,
+    mass: float = 0,
+    friction: float | None = None,
+    has_peg: bool = False,
```

Comment:
similar comment -- for now, feel free to just make `create_pybullet_triangle_with_peg`